src/cq_cam/fluent.py

```diff
@@ -13,6 +13,7 @@
     Unit,
     WorkOffset,
     WorkPlane,
+    CoolantState,
```

Comment:
isort :smiley_cat: 

Maybe I should add black & isort checks in post-commit hook. I know it's very easy to forget them!